TYPO3.Neos.NodeTypes/Resources/Private/TypoScript/Root.ts2

```diff
@@ -63,9 +63,11 @@ prototype(TYPO3.Neos.NodeTypes:MultiColumn) < prototype(TYPO3.Neos:Content) {
 	layout = ${q(node).property('layout')}
 	attributes.class = ${'container columns-' + q(node).property('layout')}
 	columns = TYPO3.TypoScript:Collection {
+		@context.columnLayout = ${q(node).property('layout')}
```

Comment:
I was not aware that repeating a FlowQuery statement has zero performance impact. Thank you for explaining!